### ISSUE TITLE
Running proteomics only analysis, error in validate_input(): ERROR. Genomics data (cna/rna) missing

### ISSUE THREAD

**Issue by sooheon** [broadinstitute/PANOPLY#45]
The PANOLY main workflow documentation says I can run analysis with just pome data, no rna/cna required. However, the entrypoint to register data into table (PANOPLY-startup-notebook) seems to expect full multiome data with cna/rna included. 

Log:
```
$$ Enter uploaded zip file name (test.zip): 240802_rsy_panoply.zip
.. proteome.gct: 1
.. groups.csv: 8
.. sample-annotation.csv: 7

============================
.. INFO. Sample annotation file successfully validated.
============================
Error in validate_input(): 
============================
.. ERROR. Genomics data (cna/rna) missing.
============================
Traceback:

1. panda_input()
2. validate_input()
3. stop(printX("ERROR", glue("Genomics data ({paste (required.genomics.types, collapse='/')}) missing")))
```

Is there a different way to register data to Workspace > DATA > TABLES?

**Comment by sooheon** [comment]
To clarify, the question is asking how to run PANOPLY on data that have different omics available than the tutorial, i.e. single omics proteome data for example. The tutorial notebook assumes a certain fileset.

**Comment by wcorinne** [comment]
In PANOPLY v1.4, this notebook restriction can be circumvent by changing the notebook to "modules" mode; in the first block of code, `panda_initialize("pipelines")` can be changed to  `panda_initialize("modules")`:
<img width="359" alt="image" src="https://github.com/user-attachments/assets/396004d0-f2e7-452a-92dd-db55e550871c">

The workflow requirement for genomics data was lifted in v1.4, but the notebook code was not updated to reflect that change, which is the reason for this error. This inconsistency has been fixed for v1.5, so switching from 'pipelines' to 'modules' mode will not be necessary in future releases of PANOPLY; v1.5 is set for public release within the week.

**Comment by sooheon** [comment]
Thanks, and congrats on a new release!
